docs/source/kernel-local.md

```diff
@@ -0,0 +1,31 @@
+## Local Mode
+
+The Local deployment is just for local development and is not meant to be run in real life operations.
+
+If you just want to try EG in a local setup, you can use the following kernelspec (no need for a launcher):
+
+```json
+{
+  "display_name": "Python 3 Local",
+  "language": "python", 
+  "metadata": {
+    "process_proxy": {
+      "class_name": "enterprise_gateway.services.processproxies.processproxy.LocalProcessProxy"
+    }
+  },
+  "argv": [
+    "python",
+    "-m",
+    "ipykernel_launcher",
+    "-f",
+    "{connection_file}"
+  ]
+}
+```
+
+`process_proxy` is optional (if Enterprise Gateway encounters a kernelspec without the `process_proxy` stanza, it will treat that kernelspec as if it contained `LocalProcessProxy`).
+
+Side note: You can run a Local kernel in [Distributed mode](./kernel-distributed.html) by setting `remote_hosts` to the localhost. Why would you do that?
+
+1. One reason is that it decreases the window in which a port conflict can occur since the 5 kernel ports are created by the launcher (within the same process and therefore closer to the actual invocation of the kernel) rather than by the server prior to the launch of the kernel process.
+2. The second reason is that auto-restarted kernels - when an issue occurs - say due to a port conflict - will create a new set of ports rather than try to re-use the same set that produced the failure in the first place. In this case, you'd want to use the [per-kernel configuration](./config-options.html#per-kernel-configuration-overrides) approach and set `remote_hosts` in the config stanza of the `process_proxy` stanza.
```

Comment:
can we append: _(so as to not prevent Standalone or YARN Client kernels from leveraging the full cluster)._

Reply:
not sure to understand... where to append `(so as to not prevent Standalone or YARN Client kernels from leveraging the full cluster).` - Do you mean Spark Standalone?

Reply:
At the end of the last sentence in bullet 2.  I want readers to understand why they'd want to update the kernel.json config stanza as opposed to `EG_REMOTE_HOSTS` (global config).